app/models/github_fetcher/user.rb

```diff
@@ -0,0 +1,27 @@
+module GithubFetcher
+  class User
+    attr_reader :token
+
+    def initialize(token)
+      @token = token
+    end
+
+    def json
```

Comment:
again, it feels like this should be standardized across all of the classes. Maybe you can inherit, like so:

```ruby
module GithubFetcher
  class Resource
    def initialize(api_path, options = {})
      @api_path = api_path
      @options = options
    end

    def as_json
      response.json_body
    end

    def response
      @response ||= GitHubBub.get(api_path, options)
    end
  end
end

module GithubFetcher
  class User < Resource
    def initialize(api_path, options)
      super
      @api_path = "/user"
    end
  end
end
```